src/manifest.json

```diff
@@ -2,10 +2,21 @@
   "manifest_version": 3,
   "name": "SmarterTabSwitching",
   "version": "0.1",
-  "icons": {
-    "16": "images/icon-16.png",
-    "32": "images/icon-32.png",
-    "48": "images/icon-48.png",
-    "128": "images/icon-128.png"
+  "permissions": ["tabs"],
+  "background": {
+    "service_worker": "background.js"
+  },
+  "action": {
+    "default_title": "Click to show an alert",
+    "default_popup": "popup.html"
+  },
+  "commands": {
+    "switch-tab": {
+      "suggested_key": {
+        "default": "Ctrl+Q"
```

Comment:
Does this mean that the keybind by default will be Ctrl + Q?

Reply:
For now. We could choose another default, if you think some other keybind would be better. Or, like I say in this issue: https://github.com/paulstn/SmarterTabSwitching/issues/25, we could try to make CTRL+TAB work, but thats much much later on the list of goals. We could replace one of our 'stretch goals' with this goal instead.

Reply:
Q sounds good to me